Modules/Core/Transform/include/itkBSplineTransform.hxx

```diff
@@ -369,6 +369,27 @@ BSplineTransform<TParametersValueType, NDimensions, VSplineOrder>
     this->m_CoefficientImages[i]->SetRegions(
       this->m_CoefficientImages[0]->GetLargestPossibleRegion() );
     }
+
```

Comment:
Is the `Conflicts:` part of the commit message intentional?

Reply:
That is the conflict I had to resolve. I guess that note is typically removed.